CuratedContent/BalterGithubCommunication.md

```diff
@@ -0,0 +1,33 @@
+[Ben Balter’s article “15 rules for communicating at GitHub”](https://ben.balter.com/2014/11/06/rules-of-communicating-at-github/)
+presents fifteen communication rules that GitHub employees follow to improve their overall workflow; rules that establish a
+communication strategy from which modern software development can greatly benefit.
+
+Resource information | Details 
+:--- | :--- 
+Resource name  | 15 rules for communicating at GitHub
+Website  | https://ben.balter.com/2014/11/06/rules-of-communicating-at-github/
+Focus | Communication in GitHub issues and PRs
+
+From his fifteen rules, here are three that stand out:
+
+  * Be flexible with tickets. Many of us think of tickets as fully completed thoughts that require time
+    and money to resolve. However, a ticket can be a question, a suggestion, an early idea that flowers
+    into a great new feature, etc. A ticket is just the start of a conversation, and software teams can
+    improve by cultivating an environment that promotes communication via tickets.
+  * Asynchronous communication means that *everyone* can get involved in the conversation. Instead of
+    engaging a single @employee, you can engage the entire team on a distributed and easy-to-track
+    platform (such as GitHub issues). This opens up the conversation to whoever on the team is willing
+    to get involved. This leads to another point:
+  * Contribute to a discussion only if you have the time to give the communication the thought it deserves.
+    That is, put your time (money) where your mouth is. If you have thoughts on an issue, make sure you have the
+    time to back them up. Otherwise, your comments can sidetrack or stall the conversation.
+    As Balter says, “Avoid drive-by opinions.”
+
+Finally, its worth reiterating that communication policies matter only if the team is dedicated to following them.
+There are many ways to break every one of Balter’s fifteen rules. For example, you can enforce PR reviews by locking
+the merge button without approval. But, this matters only if the team is dedicated to actually reviewing
+PRs instead of blindly approving (say, because of a lack of time) them. For communication policies to effect
```

Comment:
The end reads a little awkwardly. The parenthesized statement should not split the word "approving" and "them". I think it might be better written at the end of the sentence.